gps/prune.go

```diff
@@ -276,51 +272,49 @@ func isPreservedFile(name string) bool {
 	return false
 }
 
-// pruneGoTestFiles deletes all Go test files (*_test.go) within baseDir.
-func pruneGoTestFiles(baseDir string, logger *log.Logger) error {
-	files, err := collectGoTestFiles(baseDir)
-	if err != nil {
-		return errors.Wrap(err, "could not collect Go test files")
+// pruneGoTestFiles deletes all Go test files (*_test.go) in fsState.
+func pruneGoTestFiles(fsState filesystemState) error {
+	// TODO(ibrasho) detemine a sane capacity
```

Comment:
let's drop this TODO. there's really not much point in worrying about cap here, as the time for dynamic slice reallocation will be negligible relative to the time for all the `os.Remove()` system calls.